docs/content/https/tls.md

```diff
@@ -145,14 +145,62 @@ The TLS options allow one to configure some parameters of the TLS connection.
 
 !!! important "TLSOptions in Kubernetes"
 
-    When using the TLSOptions-CRD in Kubernetes, one might setup a default set of options that,
-    if not explicitly overwritten, should apply to all ingresses.  
-    To achieve that, you'll have to create a TLSOptions CR with the name `default`.
+    On Kubernetes you have to use TLSOption, a CRD in Kubernetes, with the name `default`.
     There may exist only one TLSOption with the name `default` (across all namespaces) - otherwise they will be dropped.  
+    If not explicitly overwritten, this TLSOption will apply to all ingresses.
+
     To explicitly use a different TLSOption (and using the Kubernetes Ingress resources)
     you'll have to add an annotation to the Ingress in the following form:
     `traefik.ingress.kubernetes.io/router.tls.options: <resource-namespace>-<resource-name>@kubernetescrd`
 
+### Configuration example with TLS 1.2+ and up-to-date ciphers
```

Comment:
@mloiseleur I can find those parts of the doc that are describing the configuration of a TLSOptions CRD:
https://doc.traefik.io/traefik/routing/providers/kubernetes-crd/#kind-tlsoption
https://doc.traefik.io/traefik/https/tls/#tls-options

Did you have difficulties finding those examples?